src/account/UpgradeableModularAccount.sol

```diff
@@ -238,15 +240,16 @@ contract UpgradeableModularAccount is
     }
 
     /// @notice Initializes the account with a validation function added to the global pool.
-    /// TODO: remove and merge with regular initialization, after we figure out a better install/uninstall workflow
+    /// TODO: remove and merge with regular initialization, after we figure out a better install/uninstall
+    // workflow
```

Comment:
I think it's still there, but I'm removing it on my branch anyway since it's obsolete! [37ab830](https://github.com/erc6900/reference-implementation/pull/123/commits/37ab830e353e8ac420a29d0c48d4455f7b333b9a)

Reply:
Oops sorry, probably got it mixed up with some other PRs in flight. Thanks for removing!